pom.xml

```diff
@@ -19,7 +19,9 @@
 
     <fhir.version>3.0.0</fhir.version>
     <drools.version>7.28.0.Final</drools.version>
-    <cactus-common.version>0.0.5.6</cactus-common.version>
+    <cactus-common.version>0.0.5.7</cactus-common.version>
+
+    <elastic-search-version>7.8.0</elastic-search-version>
```

Comment:
Presumably this is inherited from common, and doesn't need to be re-specified here